ios/browser/api/bookmarks/importer/imported_bookmark_entry.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_BOOKMARKS_IMPORTER_IMPORTED_BOOKMARK_ENTRY_H_
+#define BRAVE_IOS_BROWSER_API_BOOKMARKS_IMPORTER_IMPORTED_BOOKMARK_ENTRY_H_
+
+#include <vector>
+
+#include "base/strings/string16.h"
+#include "base/time/time.h"
+#include "url/gurl.h"
+
+struct ImportedBookmarkEntry {
```

Comment:
we should use the original upstream file here since it has no incompatible dependencies